Sources/StreamChat/Utils/EventBatcher.swift

```diff
@@ -34,7 +35,7 @@ final class Batcher<Item> {
     /// The timer that  calls `processor` when fired.
     private var batchProcessingTimer: TimerControl?
     /// The closure which processes the batch.
-    private let handler: ([Item]) -> Void
+    private let handler: (_ batch: [Item], _ completion: @escaping () -> Void) -> Void
```

Comment:
```suggestion
    private let handler: BatchHandler
```

Reply:
Yeah, tried that but it does not compile because the of `Batcher` type being generic